CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 13.4.8 (11/15/23)
+
+* Access lists now respect user locking. [#34619](https://github.com/gravitational/teleport/pull/34619)
+* Respect explicit deny rules for Access Requests. [#34603](https://github.com/gravitational/teleport/pull/34603)
+* Improved the error message when attempting to enroll a hardware key that cannot support passwordless. [#34590](https://github.com/gravitational/teleport/pull/34590)
```

Comment:
At some point, I'd love to see us use a consistent voice in release notes.

These first 3 bullets all take a different form:

- "Foo now does bar"
- "Do bar"
- "Made foo do bar"

Reply:
ok, I've updated it